src/App.js

```diff
@@ -28,7 +30,10 @@ export default class App extends Component {
   render() {
     return (
       <Provider store={Store}>
-        <RouterComponent />
+        <Container>
```

Comment:
i'm wondering if you can change Container for View? Its better to keep as less dependencies to externals if it is possible